---
title: DAGCircuitError (v0.28)
description: API reference for qiskit.dagcircuit.DAGCircuitError in qiskit v0.28
in_page_toc_min_heading_level: 1
python_api_type: exception
python_api_name: qiskit.dagcircuit.DAGCircuitError
---

<span id="qiskit-dagcircuit-dagcircuiterror" />

# qiskit.dagcircuit.DAGCircuitError

<Class id="qiskit.dagcircuit.DAGCircuitError" isDedicatedPage={true} github="https://github.com/qiskit/qiskit/tree/stable/0.18/qiskit/dagcircuit/exceptions.py" signature="DAGCircuitError(*msg)" modifiers="exception">
  Base class for errors raised by the DAGCircuit object.

  Set the error message.

  ### with\_traceback

  <Function id="qiskit.dagcircuit.DAGCircuitError.with_traceback" signature="with_traceback()">
    Exception.with\_traceback(tb) â€“ set self.\_\_traceback\_\_ to tb and return self.
  </Function>
</Class>

